src/utils.js

```diff
@@ -302,39 +332,21 @@ const ajaxDefaults = {
   cache: false
 }
 
-const jsonp = function(url, type, data, settings = {}) {
```

Comment:
why did you remove `jsonp`? code still use it in files directory

Reply:
Because we don't need this, it's all about small PR's, I remove $ from utils

Reply:
move it like in #690

Reply:
Ok, done